tests/performance/memory_bound_merging.xml

```diff
@@ -11,7 +11,5 @@
 
   <query>select avg(a) from remote('127.0.0.{{1,2}}', default, t_mbm) group by a format Null</query>
 
-  <query>select * from remote('127.0.0.{{1,2}}', default, t_mbm) group by a format Null settings allow_experimental_parallel_reading_from_replicas = 1, max_parallel_replicas = 2, use_hedged_requests = 0</query>
```

Comment:
Deleted this test, because Distributed table is no longer needed, but we don't have any cluster in config for performance-comparison test (cluster spec is still required)